shivatmica/41_first_missing_positive.py

```diff
@@ -1,52 +0,0 @@
-class Problem41:
-    """
-    Class for First Missing Positive (#41)
-    # https://leetcode.com/problems/first-missing-positive/
-    """
-    def firstMissingPositive(self, nums: List[int]) -> int:
```

Comment:
Break arguments into one-per-line, and add trailing comma, to preserve blame if someone changes it